src/en/news/crimson/index.md

```diff
@@ -0,0 +1,128 @@
+
+* ### PG Scaling - Splits/Merges
+
+* ### Seastore
+
+The next generation objectstore optimized for modern devices. And it would take
+a long time before its GA. As a reference, bluestore started in early 2015, and
+it was ready for production in late 2017. There will be three phases:
+
+- Prototyping/Design:
+
+  - Prototyping
+
+    - define typical use cases, the constraints and assumptions.
+    - evaluate different options and understand the tradeoffs, probably do some
+      experiments, before solidifying on a specific design
+
+  - Design:
+
+    - in-memory/on-disk data structures
+    - A sequence diagram to illustrate how to coordinate the foreground IOPS and
+      background maintenance task.
+    - define its interfaces talking to the other part of OSD
+
+- Implementation:
+
+  - Integrate the object store with crimson-osd. If seastore cannot support HDD
+    well, it should be able to coexist with bluestore.
+  - Stabilize the disk layout
+  - Dynamic disk sharding. The current sharded seastore only shards the disk space
+    statically upon mkfs (Sufficient for performance evaluation and optimization).
+  - Improve the efficiency from write-amplification and computational perspective.
+    Mostly under 4K random read and write scenarios that are CPU and I/O intensive,
+    so that switching from classic + bluestore to crimson + alienstore and later
+    on to crimson + seastore would become attractive.
+    This might be an important metric that the solution is leaving the prototyping phase.
+
+* ### Seastar+SPDK evaluation/integration
+
+- To evaluate different approaches of kernel-bypassing techniques.
+- To integrate SPDK into Seastar
+
+## Useful Project Links:
+
+- [Tracker](tracker.ceph.com/projects/crimson/issues)
+- [Docs](docs.ceph.com/en/latest/dev/crimson)
+- [Slack](ceph-storage.slack.com) - #crimson
+- [Trello](https://trello.com/b/lbHkWKxh/crimson)
+
+## Blog Posts:
+
+- [Crimson: Next-generation Ceph OSD for Multi-core Scalability](https://ceph.io/en/news/blog/2023/crimson-multi-core-scalability)
+
+## Talks:
+
+- [Cephalocon 2023 - Crimson Project Update](https://www.youtube.com/watch?v=LaP4YX1lQ3I)
+- [From Classical to the Future](https://www.youtube.com/watch?v=8N_1WAEPw0o)
+- [Ceph Virtual 2022 - What's new with Crimson and Seastore?](https://www.youtube.com/watch?v=vc5w2mn93cY)
+- [Ceph Month 2021: Crimson Update](https://www.youtube.com/watch?v=vzJPOA7aJMk)
+- [Rebuilding a Faster OSD with Future](https://www.youtube.com/watch?v=3NmCPTJI1JE)
+- [Vault '20 - Crimson: A New Ceph OSD for the Age of Persistent Memory and Fast NVMe Storage](https://www.youtube.com/watch?v=FuFmMB9rbRA)
+- [A Glimpse of the New Ceph Messenger Built on Seastar](https://www.youtube.com/watch?v=X4Pz-dqrwi8)
+- [Error handling in a futurized environment](https://www.youtube.com/watch?v=HXfFX0wGKL4)
+- [Rebuilding the Ceph Distributed Storage Solution with Seastar](https://www.youtube.com/watch?v=A_25U4j7quA)
```

Comment:
A talk might be useful:
```
[Understanding SeaStore through profiling](https://www.youtube.com/watch?v=SUJjZ9bjXJc)
```

Which is an inital attempt to understand and optimize seastore (mostly from write amplification perspective).

Reply:
Thanks!